### PR TITLE
Post release 1.3.5

### PR DESCRIPTION
**Pull request by missylbytes**
### Changes proposed in this PR ###  
Puts release/1.3.x back into dev mode after 1.3.5 release

### CODE REVIEW

File: charts/consul/values.yaml [hunk 1]
```diff
@@ -86,7 +86,7 @@ global:
   # image that is used for functionality such as catalog sync.
   # This can be overridden per component.
   # @default: hashicorp/consul-k8s-control-plane:<latest version>
-  imageK8S: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.3.5-dev
+  imageK8S: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.3-dev
```
Comment (NiniOak) [inline charts/consul/values.yaml:89]: Should this be `consul-k8s-control-plane:1.3.6-dev`?

File: charts/consul/Chart.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ annotations:
     - name: consul
       image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.17-dev
     - name: consul-k8s-control-plane
-      image: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.3.5-dev
+      image: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.3-dev
```
Comment (NiniOak) [inline charts/consul/Chart.yaml:21]: Should this be consul-k8s-control-plane:1.3.6-dev?

File: charts/consul/values.yaml (hunk 1 above)
Comment (missylbytes) [reply charts/consul/values.yaml:89]: The 1.4.x branch uses just 1.4-dev. We publish both so I think that is what we would want to do? Just use the 1.3-dev

Comment (missylbytes) [reply charts/consul/values.yaml:89]: We publish both the 1.3-dev and 1.3.6-dev

**Review by NiniOak — APPROVED** [review]

**Pull request merged**